src/ProcessBase.php

```diff
@@ -172,6 +172,7 @@ public function getOutputAsJson()
         }
         $output = preg_replace('#^[^{]*#', '', $output);
         $output = preg_replace('#[^}]*$#', '', $output);
+        $output = str_replace('\\"', '"', $output);
```

Comment:
This might be too vigorous; json might have embedded quotes that may or may not be significant. Maybe a `trim` would be safer and still sufficient?

Reply:
Looks like your new test demonstrates the correctness of this transformation. Thanks.

Reply:
Added test for single quotes, but workaround for windows cli json escaping is still in place.
I would prefer using json files.